addons/zeus/functions/fnc_moduleSetMedic.sqf

```diff
@@ -0,0 +1,53 @@
+/*
+ * Author: SilentSpike, Glowbal
+ * Assigns a medic role from the medical module to a unit
+ *
+ * Arguments:
+ * 0: The module logic <LOGIC>
+ * 1: units <ARRAY>
+ * 2: activated <BOOL>
+ *
+ * ReturnValue:
+ * nil
+ *
+ * Public: no
+ */
+
+#include "script_component.hpp"
+
+private ["_mouseOver", "_unit", "_medicN"];
+
+params ["_logic", "_units", "_activated"];
+
+if !(_activated && local _logic) exitWith {};
+
+if (isNil QEFUNC(captives,setSurrendered)) then {
```

Comment:
All looks good except from these conditions. We should probably change them all anyway to use `EFUNC(common,isModLoaded)`